src/auth0/handlers/actions.js

```diff
@@ -137,25 +193,94 @@ export default class ActionHandler extends DefaultHandler {
     }
   }
 
+
+  async createVersion(version) {
+    const actionId = version.action_id;
+    const versionToCreate = {
+      code: version.code,
+      dependencies: version.dependencies,
+      secrets: version.secrets,
+      runtime: version.runtime
+    };
+    const newVersion = await this.client.actions.createVersion({ action_id: actionId }, versionToCreate);
+
+    // wait WAIT_FOR_DEPLOY seconds for version deploy, if can't deploy an error will arise
+    await waitUntilVersionIsDeployed(this.client, actionId, newVersion.id, WAIT_FOR_DEPLOY);
+
+    // create draft version with the same values of current version
+    await this.client.actions.update({ action_id: actionId }, versionToCreate);
```

Comment:
If an action's code does not match its current version, will this overwrite the action's code?

Reply:
fixed, I removed this to leave code and current version changes independent like we talked :)